read_raw_data.R

```diff
@@ -10,17 +10,17 @@ library(readr)
 # IPUMS Data --------------------------------------------------------------
 
 #read in the IPUMS fixed-width data from gzip file.
-mydata <- read_fwf("0001_census.data.gz",
```

Comment:
`mydata` is an ambiguous name for the dataset. I changed it to `acs` so its clear which data source we are talking about.